ntcore/src/main/java/edu/wpi/first/networktables/NetworkTable.java

```diff
@@ -283,16 +366,12 @@ public boolean containsSubTable(String key) {
   public Set<String> getKeys(int types) {
     Set<String> keys = new HashSet<>();
     int prefixLen = m_path.length() + 1;
-    for (EntryInfo info : m_inst.getEntryInfo(m_pathWithSep, types)) {
+    for (TopicInfo info : m_inst.getTopicInfo(m_pathWithSep, types)) {
       String relativeKey = info.name.substring(prefixLen);
       if (relativeKey.indexOf(PATH_SEPARATOR) != -1) {
         continue;
       }
       keys.add(relativeKey);
-      // populate entries as we go
```

Comment:
Are the entries now autopopulated? How is the following code obsolete?

Reply:
We don't want to pre-emptively autopopulate.  Adding an Entry causes a subscribe, which we don't want to eagerly do.